src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -95,7 +92,26 @@ const defaultProps = {
 };
 
 class SidebarLinks extends React.Component {
-    shouldComponentUpdate(nextProps) {
+    static getUnreadReports(props) {
+        return Object.entries(props.reports)
```

Comment:
Lastly, this might be a bit trickier, but we do not use `Object.entries` in our codebase either and you can get the same functionality using some combination of the [lodash docs](https://lodash.com/docs/4.17.15). Sorry I am not able to tell you exactly what will work here right now, but I can look into it later if there will be some problems. (I would look at `_.values`.